omnipod/src/main/java/info/nightscout/androidaps/plugins/pump/omnipod/ui/OmnipodOverviewFragment.kt

```diff
@@ -510,6 +521,24 @@ class OmnipodOverviewFragment : DaggerFragment() {
         }
     }
 
+    private fun updateRileylinkStatsButton() {
+        if (omnipodManager.isRileylinkStatsButtonEnabled) {
+            omnipod_overview_button_rileylink_stats.visibility = View.VISIBLE
+            omnipod_overview_button_rileylink_stats.isEnabled = true
+        } else {
+            omnipod_overview_button_rileylink_stats.visibility = View.GONE
+        }
+    }
+
+    private fun updateTestBeepButton() {
+        if (omnipodManager.isTestBeepButtonEnabled) {
+            omnipod_overview_button_test_beep.visibility = View.VISIBLE
+            omnipod_overview_button_test_beep.isEnabled = podStateManager.isPodActivationCompleted && rileyLinkServiceData.rileyLinkServiceState.isReady && isQueueEmpty()
```

Comment:
We can actually execute this command before the pod is fully activated (see https://github.com/openaps/openomni/wiki/Command-1E-Beep-Configure#restrictions). So instead of 
`omnipod_overview_button_test_beep.isEnabled = podStateManager.isPodActivationCompleted && rileyLinkServiceData.rileyLinkServiceState.isReady && isQueueEmpty()`
We could do
`omnipod_overview_button_test_beep.isEnabled = podStateManager.isPodInitialized && podStateManager.activationProgress.isAtLeast(ActivationProgress.PAIRING_COMPLETED)
            && rileyLinkServiceData.rileyLinkServiceState.isReady && isQueueEmpty()`
like we also do for the refresh status button. That allows the user to also play the test beep when they're somewhere halfway activating a new Pod.

Reply:
Done, changed to a more general message